### PR TITLE
Add api documentation for delete rule

### PR DESCRIPTION
**Pull request by lamminade**

### CODE REVIEW

File: source/includes/azure/_security_rules.md [hunk 1]
```diff
@@ -65,3 +65,34 @@ Attributes | &nbsp;
 `sourceAddressPrefixes`<br/> *List* | List of IP address ranges or/and IP adresses. If the list is empty then all values are included.
 `destinationPortRanges`<br/> *List* | This specifies on which ports traffic will be allowed or denied by this rule. If the list is empty then all values are included.
 `destinationAddressPrefixes`<br/> *List* | List of IP address ranges or/and IP adresses. If the list is empty then all values are included.
+
+#### Get a security rule
+
+```shell 
+curl --request GET \
+  --url http://cloudmc_endpoint/v1/services/azure/co-emcilroy-eastasia/securityrules/subscriptions/6b6a1f27-55c1-4b1d-969b-60a3c9eebe64/resourceGroups/azure-ldawson-system-ldawson-azure-test/providers/Microsoft.Network/networkSecurityGroups/coldawson-test/securityRules/securityRule1 \
```
Comment (fyrnaga) [inline source/includes/azure/_security_rules.md]: obfuscate the subscription id a groups to match examples in other sections

Comment (lamminade) [reply source/includes/azure/_security_rules.md]: please refresh changes!

File: source/includes/azure/_security_rules.md [hunk 2]
```diff
@@ -65,3 +65,34 @@ Attributes | &nbsp;
...
+  --header 'mc-api-key: your_api_key'
+  ```
+
+  <code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/securityrules/:id</code>
+
+Delete an existing security rule.
+
+Query parameters | &nbsp;
+---------- | -----
+`security_group_id`<br/>*string* | The id of the [network security group](#azure-network-security-groups) in which we want to delete the security rule.
```
Comment (fyrnaga) [inline source/includes/azure/_security_rules.md]: is it mandatory or not? if so, add it to the example for the syntax

File: source/includes/azure/_security_rules.md [hunk 3]
```diff
@@ -65,3 +65,34 @@ Attributes | &nbsp;
...
+
+Query parameters | &nbsp;
+---------- | -----
+`security_group_id`<br/>*string* | The id of the [network security group](#azure-network-security-groups) in which we want to fetch the security rule.
+
+#### Delete a security rule
+
+```shell 
+curl --request DELETE \
+  --url http://cloudmc_endpoint/v1/services/azure/co-emcilroy-eastasia/securityrules/ \
```
Comment (fyrnaga) [inline source/includes/azure/_security_rules.md]: missing something here like a fake id

File: source/includes/azure/_security_rules.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/azure/_security_rules.md]: ok now

File: source/includes/azure/_security_rules.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/azure/_security_rules.md]: ok now

File: source/includes/azure/_security_rules.md (hunk 2 above)
Comment (lamminade) [reply source/includes/azure/_security_rules.md]: It is no longer mandatory

**Review by GuillaumeVialle — APPROVED** [review]

File: source/includes/azure/_security_rules.md [hunk 4]
```diff
@@ -65,3 +65,34 @@ Attributes | &nbsp;
...
+#### Get a security rule
+
+```shell
+curl --request GET \
+  --url http://cloudmc_endpoint/v1/services/azure/co-emcilroy-eastasia/securityrules/subscriptions/subscription/resourceGroups/example-system-azure-example/providers/Microsoft.Network/networkSecurityGroups/sample-network-security-group/securityRules/securityRule1 \
+  --header 'mc-api-key: your_api_key'
+  ```
+  <code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/securityrules/:id</code>
+
+Get a specific security rule by group id and rule id.
```
Comment (GuillaumeVialle) [inline source/includes/azure/_security_rules.md]: You can remove by group Id imo

File: source/includes/azure/_security_rules.md [hunk 5]
```diff
@@ -65,3 +65,26 @@ Attributes | &nbsp;
...
+
+```shell
+curl --request GET \
+  --url http://cloudmc_endpoint/v1/services/azure/co-emcilroy-eastasia/securityrules/subscriptions/subscription/resourceGroups/example-system-azure-example/providers/Microsoft.Network/networkSecurityGroups/sample-network-security-group/securityRules/securityRule1 \
+  --header 'mc-api-key: your_api_key'
+  ```
+  <code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/securityrules/:id</code>
+
+Get a specific security rule by rule id.
+
```
Comment (fyrnaga) [inline source/includes/azure/_security_rules.md:118]: No example of returned value nor document on the fields returned

**Review by fyrnaga — APPROVED** [review]
LGTM

**Pull request merged**